### ISSUE TITLE
Nomad Agitation trigger is triggering on wrong provinces

### ISSUE THREAD

**Issue by Ispil** [zijistark/EMF#633]
As related to a rare issue where Basel would become an independent nomad state, it would appear that the trigger for applying nomad agitation to a province merely specifies that it holds either no holding at all when conquered, or simply a temple. This, naturally, leads to situations where provinces with merely a bishopric (Basel being the usual example), once conquered, become a target for nomad agitation. There's also the reverse scenario, where a province that should trigger nomad agitation (several in Qazarstan in 867, where they are a single nomad holding coupled with a tribe) does not.

Additionally, when those provinces are released, characters are not properly generated- the released nomad is married to another man.

To reproduce this issue, start in any start date with the HRE ruling over Basel. Choose the Emperor of the HRE to start. Run `emf_debug.0`, and force-usurp Basel. Grant it to anyone, then `die`. Upon doing so, Basel will become an independent nomad clan.

Relevant events are `HL.4999` (the event that applies that nomad agitation modifier), and `HL.4998` (the event that actually releases the province), found in `hl_nomad_events.txt`

In short, the trigger for nomad agitation must be more robust than a simple holding check.

**Comment by zijistark** [comment]
A screenshot of it happening in Bihar:
![image](https://user-images.githubusercontent.com/6268890/53033213-6a197b80-3425-11e9-90a6-1a45c443af05.png)

**Comment by zijistark** [comment]
A screenshot of it having already happened in Basel:
![image](https://user-images.githubusercontent.com/6268890/53033280-93d2a280-3425-11e9-913e-d4cb5c6f07c8.png)

**Comment by zijistark** [comment]
A screenshot of another time w/ the homosexual marriage bit:
![image](https://user-images.githubusercontent.com/6268890/53033365-bebcf680-3425-11e9-9d6c-91644b0ea6ca.png)

**Comment by zijistark** [comment]
Closing for now. If we see this strange behavior again, please reopen issue.
